.travis.yml

```diff
@@ -7,18 +7,23 @@ env:
     - TRAVIS_JDK=11
 
 before_install:
-  - curl -sL https://raw.githubusercontent.com/shyiko/jabba/0.11.2/install.sh | bash && . ~/.jabba/jabba.sh
+  - curl -Ls https://raw.githubusercontent.com/shyiko/jabba/0.11.2/install.sh | JABBA_VERSION=0.11.2 bash && . ~/.jabba/jabba.sh
+  # Travis-CI has (as of March 2021, anyway) an outdated sbt-extras version,
+  # so overwrite it with a March 2021 version that works with sbt 1.4.8+
+  - |
+    curl -sL https://raw.githubusercontent.com/paulp/sbt-extras/dc4f350f112580fcdf5f6fa7e8d5d2116475f84a/sbt  > /tmp/sbt-launch-script || travis_terminate 1
+    chmod +x /tmp/sbt-launch-script || travis_terminate 1
+    sudo mv /tmp/sbt-launch-script /usr/local/bin/sbt || travis_terminate 1
 
-install:
-  - jabba install $(jabba ls-remote "adopt@~1.$TRAVIS_JDK.0-0" --latest=patch) && jabba use "$_" && java -Xmx32m -version
+install: jabba install "adopt@~1.$TRAVIS_JDK.0-0" && jabba use "$_" && java -Xmx32m -version
 
 stages:
   - name: validate
   - name: docs
   - name: jacoco
   - name: test
-  - name: deploy
-    if: ((branch = master AND type = push) OR (tag IS present)) AND NOT fork
+  - name: release
+    if: tag IS present
```

Comment:
I think we can keep the `if` condition. That is for publishing snapshots once PR are merged in master.

Reply:
OK!